pkg/netpol/connlist/connlist.go

```diff
@@ -47,15 +47,15 @@ type ConnlistAnalyzer struct {
 // The new interface
 // ConnlistFromResourceInfos returns the allowed-connections list from input slice of resource.Info objects,
 // and the list of all workloads from the parsed resources
-func (ca *ConnlistAnalyzer) ConnlistFromResourceInfos(info []*resource.Info) ([]Peer2PeerConnection, []Peer, error) {
+func (ca *ConnlistAnalyzer) ConnlistFromResourceInfos(info []*resource.Info) ([]Peer2PeerConnection, []Peer, []ExposedPeer, error) {
 	// convert resource.Info objects to k8s resources, filter irrelevant resources
 	objs, fpErrs := parser.ResourceInfoListToK8sObjectsList(info, ca.logger, ca.muteErrsAndWarns)
 	ca.copyFpErrs(fpErrs)
 	if ca.stopProcessing() {
 		if err := ca.hasFatalError(); err != nil {
-			return nil, nil, err
+			return nil, nil, nil, err
 		}
-		return []Peer2PeerConnection{}, []Peer{}, nil
+		return []Peer2PeerConnection{}, []Peer{}, []ExposedPeer{}, nil
```

Comment:
return `[]ExposedPeer{}` and not `nil` only if the flag for exposure analysis was true? (other places relevant)

Reply:
sure, this is the plan, I just didn't give it attention in this commit , since my goal was only to consult about the interface 
(just wrote in a way that make build will not fail)